src/styles/main.scss

```diff
@@ -0,0 +1,14 @@
+@import "./utils/variables.scss";
+@import "./utils/mixins.scss";
+@import "./blocks/clock.scss";
+
+body {
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
```

Comment:
should move from main.scss file, scss file should  consist of only imports files (blocks, variables, functions, mixins and etc)